MapleServer2/Tools/CommandManager.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Reflection;
+using MapleServer2.Commands.Core;
+using NLog;
+using Pastel;
+
+namespace MapleServer2.Tools
+{
+    public class CommandManager
+    {
+        private readonly Logger Logger = LogManager.GetCurrentClassLogger();
+
+        public static Dictionary<string, CommandBase> CommandsByAlias { get; set; }
+
+        public CommandManager()
+        {
+            CommandsByAlias = new Dictionary<string, CommandBase>();
+        }
+
+        public void RegisterAll(Assembly assembly)
+        {
+            if (assembly == null)
+            {
+                Logger.Error("Current Assembly was null.");
+                return;
+            }
+            IEnumerable<Type> callTypes = assembly.GetTypes().Where(entry => !entry.IsAbstract);
+
+            foreach (Type commandType in callTypes)
+            {
+                if (commandType.IsSubclassOf(typeof(CommandBase)))
+                {
+                    RegisterCommand(commandType);
+                    Logger.Info(commandType.Name + " was registered.");
```

Comment:
Don't think we need one print for every command, maybe add the total amount in line 40?

Reply:
This will be removed, since is a debugging log.